src/data/faq.json

```diff
@@ -542,20 +542,108 @@
+                                            { "url" : "#dcc_replacement_issues", "text": "I cannot renew a certificate in the app. What could be the reason?"},
+                                            { "url" : "#glossary_validity", "text": "Gültigkeit eines EU digitalen COVID-Zertifikats."}
+                                        ]
+                                    }
+                                ]
+                            },
+                            {
+                                "title": "Which EU digital COVID certificates are newly issued?",
+                                "anchor": "dcc_replacement_which",
+                                "active": false,
+                                "textblock": [
+                                    "The following EU digital COVID certificates are newly issued: <ul> <li>Vaccination certificate with the latest date of issue</li> <li>Recovery Certificates</li> </ul>",
+                                    "The following EU digital COVID certificates can <strong>not</strong> be exchanged: <ul> <li>Test certificates</li> <li>Blocked certificates</li> </ul>"
+                                ],
+                                "linkList": [
+                                    {
+                                        "title" : "Related FAQ articles:",
+                                        "links" : [
+                                            { "url" : "#dcc_replacement", "text": "Warum werden Zertifikate neu ausgestellt?"},
+                                            { "url" : "#dcc_replacement_howto", "text": "Wie und wann können Zertifikate in der App neu ausgestellt werden?"},
+                                            { "url" : "#dcc_replacement_vac_status", "text": "Verliere ich meinen Impfstatus wenn eines meiner Zertifikate abläuft?"},
+                                            { "url" : "#dcc_replacement_issues", "text": "I cannot renew a certificate in the app. What could be the reason?"},
+                                            { "url" : "#glossary_validity", "text": "Gültigkeit eines EU digitalen COVID-Zertifikats."}
+                                        ]
+                                    }
+                                ]
+                            },
+                            {
+                                "title": "Will I lose my vaccination status if one of my certificates expires?",
+                                "anchor": "dcc_replacement_vac_status",
+                                "active": false,
+                                "textblock": [
+                                    "Your vaccination status does not change when a certificate expires. You only need to reissue the corresponding certificate in the app to be able to prove your vaccination status via the Corona-Warn-App in the future. To do this, follow the instructions in the overview of your certificates."

```

Comment:
"You only need to reissue the corresponding certificate ..." =>
"You only need to request the reissue of the corresponding certificate ..."

RKI and the app do the reissue, not the user

Reply:
What exactly is meant with "the corresponding certificate"? The one that expires? But I don't need to reissue the certificate that expires if it is the 1/2 certificate, as this has nothing to do with the fact that I can prove my vaccination status using my 2/2 or 3/3 certificate. Please cross check.

cc @mlenkeit / @thomasaugsten

Reply:
Text from Mike is implemented, leaving it open fo @mlenkeit and @thomasaugsten to leave feedback

Reply:
@Ein-Tim The latest vaccination certificate will be reissued. E.g. if you have 1/2, 2/2, 3/3, the 3/3 will be reissued if it is about to expire. This will happen within -28 / +90 days of expiry.

Reply:
@dsarkar 

Then I suggest to change the wording a bit, because the "corresponding" certificate in this case is the certificate that expires, or am I mistaken?